javascript-practice/src/styles/layouts/toast.css

```diff
@@ -0,0 +1,28 @@
+.toastify {
+  min-width: 200px;
+  background: none;
+  text-align: center;
+}
+
+.toastify {
```

Comment:
Why do you need split this into new style instead of inside .toastify in line 1?